src/main/java/Command.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Arrays;
+
+public class Command {
+    protected String command;
+    protected int argCount;
+    protected String[] arguments;
+
+    public Command() {
+        arguments = new String[10];
+    }
+
+    public void setCommand(String command) {
+        this.command = command;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    public void addArgument(String argument) {
+        if (argCount == 9) {
+            return;
+        }
+        arguments[argCount] = argument;
+        argCount++;
+    }
```

Comment:
May I suggest 9 to be refactored as a constant? If I am not interpreting the code wrongly, I suppose it can be refactored as MAX_ARG_NUM or MAX_ARG_COUNT.